.github/workflows/build-push.yml

```diff
@@ -179,6 +191,18 @@ jobs:
         run: |
           python -m pip install cibuildwheel==2.22.0
 
+      - name: Install cython
+        run: |
+          python -m pip install 'Cython>=3.0.11,<4' 
+
```

Comment:
Should we install cython on scylladb frozen toolchain? Or will install-dependencies.sh pick up a pre-built package and so not require a compilation during installation?

Reply:
If pip would pick a correctly built wheel

You don't need todo anything on your end.

Also installing other version of cython won't help, it would pick the version mentioned in this line to build.

Its exactly what we were missing for quite some time, hence building with very old version

Reply:
I take one thing back, I'm not sure why this specific one was added, I was referring to the referring cython in setup.py

Reply:
Just, wait, it is still in progress.

Reply:
maybe this would help:

https://github.com/scylladb/python-driver/pull/412/commits/9a008b0cc7cf528c6142ce73c473f15b8a61d4e5

switching to build with build tool

let's see how CI in my PR works with it